ding/worker/collector/base_serial_evaluator.py

```diff
@@ -127,6 +128,41 @@ def update_reward(self, env_id: int, reward: Any) -> None:
             reward = reward.item()
         self._reward[env_id].append(reward)
 
+    def update_video(self, imgs):
+        for env_id, img in imgs.items():
+            if len(self._reward[env_id]) == len(self._video[env_id]):
```

Comment:
add comments for why judges new episode by this implementation

Reply:
changed